### PR TITLE
8.15.1 Release Notes

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/5493

Previews:
- [Index page](https://security-docs_bk_5748.docs-preview.app.elstc.co/guide/en/security/master/release-notes.html)
- [Release notes](https://security-docs_bk_5748.docs-preview.app.elstc.co/guide/en/security/master/release-notes-header-8.15.0.html#release-notes-8.15.1)

### CODE REVIEW

**Review by P1llus — APPROVED** [review]

File: docs/release-notes/8.15.asciidoc [hunk 1]
```diff
@@ -1,6 +1,59 @@
...
+
+
+====
+// end::known-issue-189676[]
+
+[discrete]
+[[features-8.15.1]]
+==== New features
+
+* Windows Image load events now include process protection status, making it easier to detect activities by both legitimate and malicious PPL activity.
```
Comment (gabriellandau) [inline docs/release-notes/8.15.asciidoc:28]:
```suggestion
* Windows Image load events now include process protection status, making it easier to detect both legitimate and malicious PPL activity.
```

File: docs/release-notes/8.15.asciidoc [hunk 2]
```diff
@@ -1,6 +1,59 @@
...
+==== Enhancements
+
+* Introduces several improvements to detection and parsing of log samples uploaded to automatic import ({kibana-pull}190588 [#190588 ], {kibana-pull}191502[#191502], {kibana-pull}190656[#190656], {kibana-pull}190046[#190046]).
+* Improves error handling for the Tines connector, and provides the option to use the webhook URL to connect to the Tines API ({kibana-pull}191263[#191263]).
+
+[discrete]
+[[bug-fixes-8.15.1]]
+==== Bug fixes
+
+* Provides a slight CPU usage reduction for Windows process events, especially when the same executable is repeatedly launched, such as during compilation workloads.
```
Comment (gabriellandau) [inline docs/release-notes/8.15.asciidoc:42]:
```suggestion
* Improved CPU usage for Windows process events when the same executable is repeatedly launched, such as during compilation workloads.
```

**Review by gabriellandau — COMMENTED** [review]
I didn't realize that the Defend release notes would be intermixed with non-Defend notes.  

We should probably clarify which ones pertain to Defend.  Maybe prefix them with "[Defend]" or something?

File: docs/release-notes/8.15.asciidoc [hunk 3]
```diff
@@ -1,6 +1,59 @@
...
+* Fixes an {elastic-defend} bug that sometimes caused malware scan response actions to crash when they attempted to scan an inaccessible directory. 
+* Fixes an {elastic-defend} bug that sometimes caused Endpoint to report an incorrect version if it used an independent {agent} release.
+* Fixes an {elastic-defend} bug where the `process.thread.Ext.call_stack_final_user_module.protection_provenance_path` field might be populated with a non-path value. This fix is for Windows endpoints only.
+* Fixes an {elastic-defend} bug that can lead to Endpoint reporting `STATUS_ACCESS_DENIED` when attempting to open files for `GENERIC_READ`. Endpoint almost always recovered from this issue, but with this fix, it succeeds on the first try. This fix is for Windows endpoints only.
+* Fixes an {elastic-defend} regression that was introduced in 8.13.1, where security events did not populate the `user.name` field. This fix is for Windows endpoints only.
+* Fixes an {elastic-defend} bug where Endpoint sometimes missed file and network events on newer kernels that support ebpf. This only occurs if Endpoint fails to enable ebpf probes, falling back to kprobes. This fix is for Linux endpoints only.
```
Comment (nastasha-solomon) [inline docs/release-notes/8.15.asciidoc:49]: We might need to define this acronym. @gabriellandau can you verify that the definition is correct?

```suggestion
* Fixes an {elastic-defend} bug where Endpoint sometimes missed file and network events on newer kernels that support Extended Berkeley Packet Filter (eBPF). This only occurs if Endpoint fails to enable eBPF probes, and falls back to Kprobes. This fix is for Linux endpoints only.
```

**Review by tomsonpl — DISMISSED** [review]
Jamf analyzer 👍 LGTM, thanks!

**Review by logeekal — DISMISSED** [review]
Thanks @nastasha-solomon , changes look good from investigations side.

File: docs/release-notes/8.15.asciidoc (hunk 3 above)
Comment (gabriellandau) [reply docs/release-notes/8.15.asciidoc:49]: I'm okay either way.  I think users technical enough to dig through our logs to identify the issue would know what ebpf is.  That being said, defining it here is harmless can save curious users the double click -> right click -> "Search Google"

**Review by gabriellandau — DISMISSED** [review]
LGTM.  Thanks @nastasha-solomon.

**Review by andrew-goldstein — APPROVED** [review]

File: docs/release-notes/8.15.asciidoc (hunk 3 above)
Comment (nastasha-solomon) [reply docs/release-notes/8.15.asciidoc:49]: I'll hold off on defining the acronym for now and will only commit the tense-related edits that I added above. Thanks, @gabriellandau

**Review by natasha-moore-elastic — APPROVED** [review]
Entity Analytics notes LGTM!

**Review by gabriellandau — DISMISSED** [review]

**Review by andrew-goldstein — DISMISSED** [review]

**Review by yctercero — DISMISSED** [review]
LGTM! Thanks so much!

File: docs/release-notes/8.15.asciidoc [hunk 4]
```diff
@@ -1,6 +1,59 @@
...
+* Fixes an {elastic-defend} bug where the `process.thread.Ext.call_stack_final_user_module.protection_provenance_path` field might be populated with a non-path value. This fix is for Windows endpoints only.
+* Fixes an {elastic-defend} bug that can lead to Endpoint reporting `STATUS_ACCESS_DENIED` when attempting to open files for `GENERIC_READ`. Endpoint almost always recovered from this issue, but with this fix, it succeeds on the first try. This fix is for Windows endpoints only.
+* Fixes an {elastic-defend} regression that was introduced in 8.13.1, where security events did not populate the `user.name` field. This fix is for Windows endpoints only.
+* Fixes an {elastic-defend} bug where Endpoint sometimes missed file and network events on newer kernels that support eBPF. This only occurred if Endpoint failed to enable eBPF probes and fell back to Kprobes. This fix is for Linux endpoints only.
+* Fixes a bug that caused errors to occur if you used Azure OpenAI connector for streaming ({kibana-pull}191552[#191552]).
+* Fixes a bug the prevented duplicated prebuilt rules from inheriting **Required fields** and **Related integrations** field values ({kibana-pull}191065[#191065]).
```
Comment (jmikell821) [inline docs/release-notes/8.15.asciidoc:51]:
```suggestion
* Fixes a bug that prevented duplicated prebuilt rules from inheriting **Required fields** and **Related integrations** field values ({kibana-pull}191065[#191065]).
```

File: docs/release-notes/8.15.asciidoc [hunk 5]
```diff
@@ -1,6 +1,59 @@
...
+* Turns off the option to assign users to an alert if no assignees exist ({kibana-pull}190937[#190937]).
+* Fixes a bug that prevented Timeline template settings from being applied to new Timelines that were generated by a rule ({kibana-pull}190511[#190511]).
+* Fixes an a bug that hid the option to select a connector for Elastic AI Assistant ({kibana-pull}189944[#189944]).
```
Comment (jmikell821) [inline docs/release-notes/8.15.asciidoc:54]:
```suggestion
* Fixes a bug that hid the option to select a connector for Elastic AI Assistant ({kibana-pull}189944[#189944]).
```

**Review by jmikell821 — COMMENTED** [review]
A few fixes; would like to give a quick glance again before we merge. Thanks!

File: docs/release-notes/8.15.asciidoc [hunk 6]
```diff
@@ -1,6 +1,59 @@
...
+==== New features
+
+* Introduces a new feature for {elastic-defend} where Windows Image load events now include process protection status, making it easier to detect both legitimate and malicious PPL activity.
+* Allows Jamf data to be examined in the visual event analyzer ({kibana-pull}190965[#190965]).
+
+[discrete]
+[[enhancements-8.15.1]]
+==== Enhancements
+
+* Improves {elastic-defend} by reducing Malware Protection I/O and CPU usage when recently-written are subsequently executed. This change is for Windows endpoints only.
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/release-notes/8.15.asciidoc:35]: Thanks, both! @gabriellandau I'm making one a minor edit to your suggested revision below to remove the dash after "recently" (that was my fault). Everything else looks good, thanks!

```suggestion
* Improves {elastic-defend} by reducing Malware Protection I/O disk and CPU usage when recently written files are subsequently executed. This update is for Windows endpoints only.
```

[the remark this answers is not in the document]
Comment (gabriellandau) [reply docs/release-notes/8.15.asciidoc]: We need to put `disk` before `I/O`.

File: docs/release-notes/8.15.asciidoc [hunk 7]
```diff
@@ -1,6 +1,59 @@
...
+==== New features
+
+* Introduces a new feature for {elastic-defend} where Windows Image load events now include process protection status, making it easier to detect both legitimate and malicious PPL activity.
+* Allows you to examine Jamf data in the visual event analyzer ({kibana-pull}190965[#190965]).
+
+[discrete]
+[[enhancements-8.15.1]]
+==== Enhancements
+
+* Improves {elastic-defend} by reducing Malware Protection I/O disk and CPU usage when recently written files are subsequently executed. This update is for Windows endpoints only.
```
Comment (gabriellandau) [inline docs/release-notes/8.15.asciidoc:35]:
```suggestion
* Improves {elastic-defend} by reducing Malware Protection disk I/O and CPU usage when recently written files are subsequently executed. This update is for Windows endpoints only.
```

File: docs/release-notes/8.15.asciidoc (hunk 6 above)
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/release-notes/8.15.asciidoc]: Oops, I accidentally pasted in the wrong order. Thank you - fixed!

File: docs/release-notes/8.15.asciidoc [hunk 8]
```diff
@@ -1,6 +1,59 @@
...
+
+[discrete]
+[[bug-fixes-8.15.1]]
+==== Bug fixes
+
+* Fixes an {elastic-defend} bug that affected CPU usage for Windows process events where the same executable is repeatedly launched, for example, during compilation workloads. With this fix, CPU usage is improved.
+* Fixes an {elastic-defend} bug that sometimes caused malware scan response actions to crash when they attempted to scan an inaccessible directory. 
+* Fixes an {elastic-defend} bug that sometimes caused {elastic-endpoint} to report an incorrect version if it used an independent {agent} release.
+* Fixes an {elastic-defend} bug where the `process.thread.Ext.call_stack_final_user_module.protection_provenance_path` field might be populated with a non-path value. This fix is for Windows endpoints only.
+* Fixes an {elastic-defend} bug that can lead to Endpoint reporting `STATUS_ACCESS_DENIED` when attempting to open files for `GENERIC_READ`. Endpoint almost always recovered from this issue, but with this fix, it succeeds on the first try. This fix is for Windows endpoints only.
```
Comment (jmikell821) [inline docs/release-notes/8.15.asciidoc:47]:
```suggestion
* Fixes an {elastic-defend} bug that can lead to {elastic-endpoint} reporting `STATUS_ACCESS_DENIED` when attempting to open files for `GENERIC_READ`. {elastic-endpoint} almost always recovered from this issue, but with this fix, it succeeds on the first try. This fix is for Windows endpoints only.
```

File: docs/release-notes/8.15.asciidoc [hunk 9]
```diff
@@ -1,6 +1,59 @@
...
+* Fixes an {elastic-defend} regression that was introduced in 8.13.1, where security events did not populate the `user.name` field. This fix is for Windows endpoints only.
+* Fixes an {elastic-defend} bug where {elastic-endpoint} sometimes missed file and network events on newer kernels that support eBPF. This only occurred if {elastic-endpoint} failed to enable eBPF probes and fell back to Kprobes. This fix is for Linux endpoints only.
+* Fixes a bug that caused errors to occur if you used Azure OpenAI connector for streaming ({kibana-pull}191552[#191552]).
```
Comment (jmikell821) [inline docs/release-notes/8.15.asciidoc:50]:
```suggestion
* Fixes a bug that caused errors if you used Azure OpenAI connector for streaming ({kibana-pull}191552[#191552]).
```

**Review by jmikell821 — COMMENTED** [review]
Two small nits, then this will be good to merge.

**Review by joepeeples — APPROVED** [review]

**Pull request merged**